docker/docker-entrypoint.sh

```diff
@@ -39,10 +39,14 @@ if not User.objects.filter(username='${SUPERUSER_NAME}'):
     Token.objects.create(user=u, key='${SUPERUSER_API_TOKEN}')
 END
 
-for script in /opt/netbox/startup_scripts/*.py; do
-  echo "⚙️ Executing '$script'"
-  ./manage.py shell --interface python < "${script}"
-done
+RUN_STARTUP_SCRIPTS=${RUN_STARTUP_SCRIPTS:="true"}
+if [ "$RUN_STARTUP_SCRIPTS" = "true" ]
```

Comment:
I would suggest to do it the other way round:

```bash
if [ "$SKIP_STARTUP_SCRIPTS" == "true"]; then
```

This would completely eliminate the need for the default variable assignment.

Reply:
Also, to match the style of other scripts (like [build-latest.sh][build-latest.sh]), please use `==` instead of `=` and put the `then` on the same line as the `if`.

[build-latest.sh]: https://github.com/netbox-community/netbox-docker/blob/master/build-latest.sh#L66